test/functional/wallet_import_rescan.py

```diff
@@ -20,28 +20,38 @@
 """
 
 from test_framework.test_framework import BitcoinTestFramework
+from test_framework.address import AddressType
 from test_framework.util import (
     connect_nodes,
     assert_equal,
     set_node_times,
 )
 
 import collections
+from decimal import Decimal
```

Comment:
this import is added in the wrong commit. It needs to be added in _test: Replace fragile "rng" with call to random()_

Reply:
Fixed